float32[] pose_around_chair
---
---